lambdas/death-enrichment/src/main/java/uk/gov/di/data/lep/DeathEnrichment.java

```diff
@@ -55,7 +56,13 @@ private DeathNotificationSet enrichData(GroJsonRecordWithCorrelationID baseData)
 
     @Tracing
     private void audit(DeathNotificationSet enrichedData) {
-        var auditDataExtensions = new DeathEnrichmentAuditExtensions(enrichedData.hashCode(), enrichedData.txn());
-        addAuditDataToQueue(new DeathEnrichmentAudit(auditDataExtensions));
+        try {
+            var enrichedDataHash = Hasher.hash(objectMapper.writeValueAsString(enrichedData));
```

Comment:
I'm not sure I'm a massive fan of this approach (as we're writing the value as string twice), but hashing an object seemed more fiddly and maybe the less correct way of doing it?